gpii/node_modules/gpii-oauth2/gpii-oauth2-utilities/src/OAuth2Utilities.js

```diff
@@ -17,6 +17,44 @@ var fluid = fluid || require("infusion");
 var gpii = fluid.registerNamespace("gpii");
 fluid.registerNamespace("gpii.oauth2");
 
+gpii.oauth2.errors = {
```

Comment:
You could supply fluid.freezeRecursive to it here to convince yourself you have not made an error in gpii.oauth2.composeError :)